src/sanitizer.js

```diff
@@ -8,18 +8,67 @@
 // eslint-disable-next-line no-unused-vars
 export const _sanitizeDocFragment = function _sanitizeDocFragment(config, input) {
     // TODO implement this function and remove eslint-disable comment above
+    // TODO maybe we can use https://developer.mozilla.org/en-US/docs/Web/API/NodeIterator to traverse through nodes
 };
 
 // eslint-disable-next-line no-unused-vars
 export const _sanitizeDocument = function _sanitizeDocument(config, input) {
     // TODO implement this function and remove eslint-disable comment above
 };
 
-// a parser that doesn't run scripts or loads resources
-// eslint-disable-next-line no-unused-vars
+/**
+ * fragmentParser
+ * a parser that doesn't run scripts or loads resources
+ *
+ * @param {Node} ctx DOM node
+ * @param {Object} input Document or DocumentFragment
```

Comment:
Thanks for for adding function signature documentation. I filed #7 to ensure we get this consitently.

`input` should be a string btw!

Reply:
> Thanks for for adding function signature documentation. I filed mozilla#7 to ensure we get this consitently.

Oh, that's a good idea! I'll have another PR up soon to add [`eslint-plugin-jsdoc`](https://github.com/gajus/eslint-plugin-jsdoc) rules to help that. I figured we try to use `eslint-plugin-jsdoc` instead of what is packaged with ESLint, because [`require-jsdoc` is EOL](https://eslint.org/blog/2018/11/jsdoc-end-of-life)

> input should be a string btw!

Oh, thanks for catching that! :) If `input` is a string, that means we can throw out everything in this function and just use [this code block](https://github.com/0xedward/sanitizer-polyfill/pull/1/files#diff-d64aaabdffa3cd9703c8597141f114de6a2cd97f9cb8a2fda3bc09aa1fbd148bR67-R71) instead. I'll update the PR with this new info

Reply:
Go for it :)